configs/paper_cfgs/ssdnerf_cars_uncond.py

```diff
@@ -1,5 +1,14 @@
+import os
 name = 'ssdnerf_cars_uncond'
 
+data_path = '/net/tscratch/people/plgmarzol/SSDNeRF_data/shapenet'
+
+cache_path = os.getcwd()
+cache_path = os.path.dirname(cache_path)
+cache_path = os.path.join(cache_path, "SSDNeRF_data", "shapenet")
+if not os.path.exists(cache_path):
```

Comment:
Please make sure that the cache_path has write permissions for all of us. This will help to maintain a shared code baseline.